insta-bot/instabot.py

```diff
@@ -0,0 +1,89 @@
+from selenium import webdriver
+from selenium.webdriver.chrome.options import Options
+from selenium.common.exceptions import NoSuchElementException
+from pandas import read_csv
+from time import sleep
+
+
+def getCredentials():
```

Comment:
As you have deleted the non required files and adapted the JSON format, don't you think all these functions need to be updated?

Reply:
Sorry for it. It was my mistake I should have updated it